### PR TITLE
ROC-6039 Add dashboard for free mediation accepted by both parties

### PR DESCRIPTION
**Pull request by simad**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-6039

### Change description

Adding new dashboard statuses and claim details if both parties have said yes to free mediation.

### Work checklist

- [x] Unit tests added where applicable
- [x] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/test/features/dashboard/routes/dashboard/full-defence.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ import { FreeMediationOption } from 'forms/models/freeMediation'
 import {
   baseDefenceData,
   baseResponseData,
-  defenceWithAmountClaimedAlreadyPaidData
+  defenceWithAmountClaimedAlreadyPaidData, defenceWithDisputeData
```
Comment (sudheer-ajax) [inline src/test/features/dashboard/routes/dashboard/full-defence.ts]: probably goes into a new line

File: src/test/features/dashboard/routes/index.ts [hunk 2]
```diff
@@ -23,9 +23,10 @@ import {
   basePayImmediatelyData,
   baseResponseData,
   defenceWithAmountClaimedAlreadyPaidData,
-  partialAdmissionAlreadyPaidData
+  partialAdmissionAlreadyPaidData, partialAdmissionWithPaymentBySetDateData
```
Comment (sudheer-ajax) [inline src/test/features/dashboard/routes/index.ts]: please add the new import into new line

Comment (simad) [reply src/test/features/dashboard/routes/index.ts]: 👍

File: src/test/features/dashboard/routes/dashboard/full-defence.ts (hunk 1 above)
Comment (simad) [reply src/test/features/dashboard/routes/dashboard/full-defence.ts]: 👍

**Pull request closed without merging**